app/controllers/api/v1/users_controller.rb

```diff
@@ -60,8 +60,23 @@ def orders_count
         render json: Order.counts_for(params[:id])
       end
 
+      def mentionable_users
+        @users = Messages::Channels
+                 .new(app_name: app_name,
+                      messageable: messageable_obj,
+                      is_private: params[:is_private],
+                      current_user: current_user)
+                 .related_users
+        render json: { users: @users }
+      end
+
       private
 
+      def messageable_obj
+        return Order.find(params[:order_id]) if params[:order_id].present?
+        return Offer.find(params[:offer_id]) if params[:offer_id].present?
```

Comment:
If no params are given, this would return nil. Could you make sure there's a test for this scenario :)